bowtie/templates/report.html.j2

```diff
@@ -2,6 +2,9 @@
 <html lang="en">
 
 <style>
+  a:link{
+    text-decoration: none;
```

Comment:
Here too -- can you elaborate on which link you're adding this for? I would also suspect Bootstrap should have some class for this though I didn't check.

Reply:
Currently There are no need to use `<a href="#">` for these links schema and test name, I just wanted to remove default link style (Underline)
![Screenshot_90](https://user-images.githubusercontent.com/92979541/231989777-d11ca1d6-b43e-4556-8ff6-21bfabdcb707.png)

Reply:
Done! Used bootstrap class instead of hard coded css